exes/mqtt-broadcaster/inc/mqtt_broadcaster.hpp

```diff
@@ -363,9 +412,28 @@ class mqtt_broadcaster {
 
     if (payload.has_timestamp() && !payload.has_seq() &&
         (publish_packet.opts().get_qos() == async_mqtt::qos::at_most_once) &&
-        (publish_packet.opts().get_retain() == async_mqtt::pub::retain::no) && (metric.name() == rebirth_metric)) {
-      logger_.trace("Conditions met. Sending NBIRTH...");
-      co_await asio::co_spawn(mqtt_client_->strand(), send_nbirth(), asio::use_awaitable);
+        (publish_packet.opts().get_retain() == async_mqtt::pub::retain::no)) {
+      if (metric.name() == rebirth_metric) {
+        logger_.trace("Conditions met. Sending NBIRTH...");
+        co_await asio::co_spawn(mqtt_client_->strand(), send_nbirth(), asio::use_awaitable);
+      } else {
+        logger_.trace("Metric received. Checking conditions...");
+        payload.PrintDebugString();
```

Comment:
what does this do: `payload.PrintDebugString();`

Reply:
Print the payload, switching these statements out for:

```cpp
logger_.trace("Payload: \n {}", payload.DebugString());
```